common/config/azure-pipelines/jobs/regression-testing.yaml

```diff
@@ -17,33 +17,109 @@ schedules:
         - release/*
 
 jobs:
-  - template: ci-core.yaml
-    parameters:
-      name: Node_18
-      nodeVersion: 18.x
-      pool:
-        demands: Agent.OS -equals $(platform)
-  - template: ci-core.yaml
-    parameters:
-      name: Node_16
-      nodeVersion: 16.x
-      pool:
-        demands: Agent.OS -equals $(platform)
-  - template: ci-core.yaml
-    parameters:
-      name: Node_14_x
-      nodeVersion: 14.x
-      pool:
-        demands: Agent.OS -equals $(platform)
-  - template: ci-core.yaml
-    parameters:
-      name: Node_12
-      nodeVersion: 12.x
-      pool:
-        demands: Agent.OS -equals $(platform)
-  - template: ci-core.yaml
+- job:
+  variables:
+  - name: linuxPool
+    value: 'iModeljs Linux'
+  - name: mac_pool
+    value: iModelTechCI
+  - name: win_pool
+    value: iModelTechCI
+
+  strategy:
+    matrix:
```

Comment:
With adding all of these to the matrix, can you limit the number in parallel? I think before it's run each set of jobs based on Node versio with a max number of concurrent jobs of 3.